lib/binance.ex

```diff
@@ -35,6 +35,46 @@ defmodule Binance do
     end
   end
 
+  @doc """
+  Get historical trades
+  https://binance-docs.github.io/apidocs/spot/en/#old-trade-lookup
+
+  Returns `{:ok, [%Binance.HistoricalTrade{}]}` or `{:error, reason}`.
+
+  ## Example
+  {:ok,
+    [%Binance.HistoricalTrade{
+     id: 192180149,
+     is_best_match: true,
+     is_buyer_maker: true,
+     price: "1.79878000",
+     qty: "55.50000000",
+     quote_qty: "99.83229000",
+     time: 1618341167715
+   }]
+  }
+  """
+  def get_historical_trades(symbol, limit)
+      when is_binary(symbol) and is_integer(limit) do
+    params = %{
+      symbol: symbol,
+      limit: limit
+    }
+
+    request_historical_trades(params)
+  end
+
+  def get_historical_trades(symbol, limit, fromId)
+      when is_binary(symbol) and is_integer(limit) and is_integer(fromId) do
+    params = %{
+      symbol: symbol,
+      limit: limit,
+      fromId: fromId
+    }
+
+    request_historical_trades(params)
```

Comment:
would it be possible to compress those two functions into one with optional `fromId` handled in a similar fashion as in other places in the Binance module:

```
    arguments =
      %{
        symbol: symbol,
        limit: limit
      }
      |> Map.merge(
        unless(
          is_nil(from_id),
          do: %{fromId: from_id},
          else: %{}
        )
      )
```

Also, could you please use snake_case style for variable names?

Reply:
DONE